src/vm/runners/cairo_runner.rs

```diff
@@ -334,6 +337,53 @@ impl CairoRunner {
         vm.segments.get_memory_holes(&builtin_accessed_addresses)
     }
 
+    pub fn end_run(
+        &mut self,
+        _disable_trace_padding: Option<bool>,
+        disable_finalize_all: Option<bool>,
```

Comment:
These `Option<_>`s look weird. Why not bare `bool`?

Reply:
In Python there's a default value, which can't be done in Rust.

Reply:
Then it makes no sense to attempt it. The point of default parameters is that you don't need to pass one. Here, you need to always pass one, it's now just more inconvenient since my "default" case is, rather than passing what I wanted, passing `None`, but when I want to be explicit now I need to do `Some<truth_value>`.

Reply:
Note we do accept this idiom in `cairo-rs-py` because PyO3 knows to consider `Option<_>` arguments in pymethods as optional arguments inside of Python.